tests/data/group_system/group_network/group_network-ipv6/group_disabling_ipv6/rule_network_ipv6_disable_rpc/rpc_disabled.pass.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+#
+# profiles = xccdf_org.ssgproject.content_profile_ospp-rhel7, xccdf_org.ssgproject.content_profile_ospp-rhel7
+# remediation = bash
+
+sed -i "/^tcp6[[:space:]]\+tpi\_.*inet6.*/d" /etc/netconfig
```

Comment:
What is the escaped underscore for? I haven't been able to find anything about it.

Reply:
I actually don't know. I sourced from the remediation script itself. Will try to look into it.

Reply:
In fact there is no need to escape underscores, removed the backslash.